CIP-1855/README.md

```diff
@@ -25,7 +25,8 @@ HD       | Hierarchical Deterministic, refers to wallets as described in [BIP-00
 
 ## Overview 
 
-Forging tokens is derived from a script policy. The script policy includes hashes of keys needed to forge new tokens and must be witnessed by the keys with hashes listed. 
+Forging tokens is derived from a script policy. The script policy includes hashes of keys needed to forge new tokens and must be witnessed by the keys with hashes listed.
```

Comment:
This isn't quite accurate. It can require all keys in the policy, or it can have thresholds of number of keys, such as a 2/3 or 3/5 policy.